### ISSUE TITLE
!store not working

### ISSUE THREAD

**Issue by ghost** [alongubkin/store#19]
i followed all the steps but !store !shop !inv !gift and every other commands is not working
what should i do? i already did a web panel. btw i saw a thread saying that theres a smx file.. tried finding it but i cudnt find it.. is it important?

**Comment by Chdata** [comment]
The instructions are bit out of date and what you need to do isn't really clear cut because for some things you even have to mix together the alpha and developer version. Like the gift part of the plugin, I think only the alpha version works properly and the developer one is messed up, but iirc I use most of the developer versions of everything.

**Comment by Arrow768** [comment]
@danialsaufi Yes, the SMX Files are important, the SP files alone wont do it.
First of all, you should tell use the game you are trying to use with store (CS:S, CS:GO, TF2, ...) and the Sourcemod Version you are using

**Comment by ghost** [comment]
@Arrow768 im using the store for cs go.. second im using the latest version of sourcemod.third, where can i download the smx files? and lastly i already put the web panel. when i go to the webpanel theres a line that says common/core/loader.php on top of every page.but i can still login and manage and everything just that theres that thing on top. so i was wondering will it affect anything? im using 5.6 php.  sorry this is my first time using a plugin so im kinda not used to this yet.:) ty for replying arrow

**Comment by ghost** [comment]
@Chdata hmm so which one should i use? they say the alpha is buggy and the new one is fixed. got any ideas on how? total newbie here so im not used to this yet
